drivers/hwinfo/hwinfo_mcux_src.c

```diff
@@ -115,6 +121,10 @@ int z_impl_hwinfo_get_supported_reset_cause(uint32_t *supported)
 #if (defined(FSL_FEATURE_SRC_HAS_SCR_LOCKUP_RST) && FSL_FEATURE_SRC_HAS_SCR_LOCKUP_RST)
 		      | RESET_CPU_LOCKUP
 #endif
+#if (defined(FSL_FEATURE_SRC_HAS_SRSR_LOCKUP_SYSRESETREQ) && \
+     FSL_FEATURE_SRC_HAS_SRSR_LOCKUP_SYSRESETREQ)
```

Comment:
Hi @Cladoc- it looks like the compliance checks are flagging this line. You likely need to replace the leading spaces here with a tab for indentation.

Also, could you shorten your commit title slightly? Something like `drivers: src: hwinfo: Report reset cause for lockup_sysresetreq` should work, and won't cause the title to overflow into the commit body (as it currently does, at least in the github UI)

Reply:
Hi @danieldegrasse, I replaced the title and offending spaces.